app/controllers/submissions_controller.rb

```diff
@@ -120,6 +136,28 @@ def destroy
                                                    @submission.assessment)) && return
   end
 
+  action_auth_level :destroyBatch, :instructor
+  def destroyBatch
```

Comment:
nit: better to use snake case instead of camel case for controller methods; make sure you update the path variable name in the frontend too

Reply:
fixed.